sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/CallMediaRecognitionType.java

```diff
@@ -13,6 +13,10 @@ public final class CallMediaRecognitionType extends ExpandableStringEnum<CallMed
     public static final CallMediaRecognitionType DTMF = fromString("dtmf");
     /** Static value choices for RecognitionTypeInternal. */
     public static final CallMediaRecognitionType CHOICES = fromString("choices");
+    /** Static value continuous speech for RecognitionType. */
+    public static final CallMediaRecognitionType SPEECH = fromString("speech");
+    /** Static value continuous speech or dtmf for RecognitionTypeInternal. */
+    public static final CallMediaRecognitionType SPEECHORDTMF = fromString("speechordtmf");
```

Comment:
> The names of variables declared class constants and of ANSI constants should be all uppercase with words separated by underscores ("_")

https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

Reply:
changed to underscore